script.js

```diff
@@ -1,22 +1,21 @@
-function getInterestingFields(contentItem){
-    //console.log(contentItem);
-    var mainImage = getImage(contentItem);
+function getInterestingFields(content){
+    const mainImage = getImage(content);
     return {
-        headline: contentItem.fields.headline,
-        trailText: contentItem.fields.trailText,
-        mainImage: mainImage ? mainImage : contentItem.fields.thumbnail
+        headline: content.fields.headline,
+        trailText: content.fields.trailText,
+        mainImage: mainImage
     }
 }
 
 function addRowOfNews(rowSubject, numberOfItems){
-    var contentApiQuery = encodeURIComponent(rowSubject);
-    return addContentList(rowSubject, "http://content.guardianapis.com/search?q="+contentApiQuery, numberOfItems);
+    const contentApiQuery = encodeURIComponent(rowSubject);
+    return addContentList(rowSubject, `http://content.guardianapis.com/search?q=${contentApiQuery}`, numberOfItems);
```

Comment:
Template strings [aren't supported in IE](https://caniuse.com/#feat=template-literals). Is it safe to assume the page will be viewed in a modern browser?